app/src/main/java/org/glucosio/android/activity/AddGlucoseActivity.java

```diff
@@ -74,13 +78,13 @@ protected void onCreate(Bundle savedInstanceState) {
         setPresenter(presenter);
```

Comment:
Im a bit confused why a presenter is used if this class has logic, also there is no view in the presenter, so it depends on the activity, making it impossible to unit test

Reply:
Kinda of true, but I'm not going to do anything with it now